### ISSUE TITLE
Allow alt text for images

### ISSUE THREAD

**Issue by doom-cat** [doom-cat/PHPWord-v1.1.0#1]
Users of PhpWord should be able to include alt text associated with an added image.

**Comment by doom-cat** [comment]
Closed with commits called "Add image alt text".
